composer.json

```diff
@@ -3,9 +3,8 @@
     "description": "Elastic Driver for Laravel Scout",
     "keywords": ["laravel", "scout", "elasticsearch", "elastic"],
     "require": {
-        "php": ">=5.6.4",
-        "laravel/scout": "^3.0|^4.0",
-        "elasticsearch/elasticsearch": "^5.0"
+        "laravel/scout": "^5.0",
+        "elasticsearch/elasticsearch": "^6.0"
```

Comment:
Is upgrading to ES-PHP 6 really required for Laravel 5.6 / Scout 5? Maybe this should be a seperate PR.